### PR TITLE
New Header

### LINKED ISSUE #49: Header Updates
- [x] Update the style guide header first
- [x] Headers section should highlight the one that is in the viewport
- [x] Smooth scroll (via `scroll-behavior`)

### PR DESCRIPTION
**Pull request by wilwade**
# Description

Updates the context around the new header so that it works correctly for the site

- Closes #49 

## Screenshot(s)

image

image

image

### CODE REVIEW

File: src/components/Header/HeaderNav.svelte
Comment (wilwade) [inline src/components/Header/HeaderNav.svelte:1]: These files are no longer used and now part of the style-guide

File: src/components/Sections/SectionWrapper.svelte [hunk 1]
```diff
@@ -3,6 +3,6 @@
   export let classes = '';
 </script>
 
-<div {id} class={`w-full ${classes}`}>
+<div {id} class={`scroll-target w-full ${classes}`}>
```
Comment (wilwade) [inline src/components/Sections/SectionWrapper.svelte:6]: Needed to make sure that the scroll margin is set correctly

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
-  <Header />
-</div>
-<div class="max-w-screen-xl">
+<Header />
+<div class="w-full">
   <slot />
 </div>
-<Footer type="dark" class="w-full border-grayBorder border-opacity-25 bg-navy vertical-lg:w-[100%]" />
+<div class="w-full bg-navy px-[5%] lg:px-[20%]">
+  <Footer type="dark" class="w-full border-grayBorder border-opacity-25 vertical-lg:w-[100%]" />
```
Comment (wilwade) [inline src/routes/+layout.svelte:11]: Allows the footer to have a full width background color along with the desired padding from the desgin

File: src/style/app.css [hunk 3]
```diff
@@ -66,9 +66,17 @@
     animation: slide-from-right 1s ease-out;
     -webkit-animation: slide-from-right 1s ease-out;
   }
+
+  .scroll-target {
+    scroll-margin-top: 90px;
```
Comment (wilwade) [inline src/style/app.css:71]: Header size + 5px

**Review by mattheworris — APPROVED** [review]
- Read through changes
- Clicked around preview

🚢 it!!

**Pull request merged**